_posts/2019-01-09-cockpit-185.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Cockpit 185
+author: Gundersanne
+date: 2019-01-09
+tags: cockpit linux
+slug: cockpit-185
+category: release
+summary: Responsive dialogs and kubernetes containers included in docker page graphs
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 185.
+
+### Responsive dialogs
+
+Dialogs on the network page and kdump page are now responsive. As are the
+password dialogs on the users page.
```

Comment:
How about "Dialogs on the Networking and KDump pages, as well as the password dialogs on the Accounts page are now responsive"? This avoids repetitions  and uses their actual menu entry names.